### ISSUE TITLE
[BITV] 9.4.1.2/4.2 - The status of the button "Add" is only communicated to screen reader users in the "expanded" state using the aria-expanded="true" attribute. The "aria-expanded" attribute should be used in both states. (1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41897]
https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v4-n2

**Comment by ShGKme** [comment]
The issue is on the `NcActions` side.

But the issue is not clear.

All our menus are not present on the page until they are opened by clicking on the button. So we cannot have `aria-expanded="false"` because we cannot add `aria-controls`. It requires the menu itself to be presented on the page, just hidden.

Currently we remove `aria-expanded` when it is not expanded according to the W3C recommendations.

> The aria-expanded attribute is removed when the menu is closed.

Link: https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-actions 

However, there is also a discussion in w3c about changing recommendations here: https://github.com/w3c/aria-practices/pull/2829

**Comment by ShGKme** [comment]
> Currently we remove `aria-expanded` when it is not expanded according to the W3C recommendations.
> 
> > The aria-expanded attribute is removed when the menu is closed.
> 
> Link: https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-actions
> 
> However, there is also a discussion in w3c about changing recommendations here: [w3c/aria-practices#2829](https://github.com/w3c/aria-practices/pull/2829)

Example on [w3c patterns](https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-actions/) says "The aria-expanded attribute is removed when the menu is closed.".

However there is an ongoing discussion about it.
For example, PR 2829 on https://github.com/w3c/aria-practices/pull/. Or another example https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/

So actually it is recommended to add `aria-expanded="false"`, not remove the attribute.
